src/auto/desktop_app_info.rs

```diff
@@ -91,10 +94,14 @@ pub trait DesktopAppInfoExt: 'static {
         launch_context: Option<&P>,
     );
 
-    //fn launch_uris_as_manager<P: IsA<AppLaunchContext>>(&self, uris: &[&str], launch_context: Option<&P>, spawn_flags: /*Ignored*/glib::SpawnFlags, user_setup: Option<Box_<dyn FnOnce() + 'static>>, pid_callback: /*Unimplemented*/FnMut(&DesktopAppInfo, /*Ignored*/glib::Pid), pid_callback_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Result<(), Error>;
-
-    //#[cfg(any(feature = "v2_58", feature = "dox"))]
-    //fn launch_uris_as_manager_with_fds<P: IsA<AppLaunchContext>>(&self, uris: &[&str], launch_context: Option<&P>, spawn_flags: /*Ignored*/glib::SpawnFlags, user_setup: Option<Box_<dyn FnOnce() + 'static>>, pid_callback: /*Unimplemented*/FnMut(&DesktopAppInfo, /*Ignored*/glib::Pid), pid_callback_data: /*Unimplemented*/Option<Fundamental: Pointer>, stdin_fd: i32, stdout_fd: i32, stderr_fd: i32) -> Result<(), Error>;
+    fn launch_uris_as_manager<P: IsA<AppLaunchContext>>(
+        &self,
+        uris: &[&str],
+        launch_context: Option<&P>,
+        spawn_flags: glib::SpawnFlags,
+        user_setup: Option<Box_<dyn FnOnce() + 'static>>,
```

Comment:
Not sure this is correct or safe. The function is called once per process, so if you have multiple URIs... :) OTOH this is called right after `fork()` from what I understand. Unclear if `FnOnce` is correct in this case.

Reply:
Also unclear if Rust is supposed to be fork-safe, or if the closure needs to be marked unsafe anyway